### ISSUE TITLE
Error occurs in fontawesome 0.4.0 installed environment

### ISSUE THREAD

**Issue by choonghyunryu** [choonghyunryu/dlookr#87]
If fontawesome package version 0.4.0 or higher is installed, the following error occurs in the report generation functions.

````
pandoc: /Library/Frameworks/R.framework/Versions/4.2-arm64/Resources/library/fontawesome/fontawesome/css/../webfonts/fa-v4compatibility.woff2: openBinaryFile: does not exist (No such file or directory)
Error: pandoc document conversion failed with error 1
````

The function that is causing the problem is:

- diagnose_web_report()
- eda_web_report()

**Comment by choonghyunryu** [comment]
I forwarded the issue to the fontawesome developer on github.

https://github.com/rstudio/fontawesome/issues/99

**Comment by the8thday** [comment]
i have the same problem, and thanks for your forward!

**Comment by choonghyunryu** [comment]
The problem has been fixed in version 0.5.0 (2023-01-25) of fontawesome on CRAN. So close the issue.
